aws-transfer-agreement/.rpdk-config

```diff
@@ -0,0 +1,28 @@
+{
+    "artifact_type": "RESOURCE",
+    "typeName": "AWS::Transfer::Agreement",
+    "language": "java",
+    "runtime": "java8",
```

Comment:
Are we using java8 or java11 for the project? We are currently using java11 for workflows

Reply:
We are using java11 too, updating it.